### DISCUSSION TITLE
Logging level for "request completed" message

### DISCUSSION THREAD

**Discussion by duel-adn** [directus/directus#11566] [Archive]
Hello,
not sure this is the right place for this proposal, it is of minor importance, but useful IMO.

When directus is installed in a production environment with health checking turned on, it logs the request to the `/server/ping` path, adding noise to the logs.

My proposal would be to downgrade the log level of this particular request, or to downgrade the log level of all the ` request completed GET 200 <PATH> <TIME>ms`, as they don't give that much of information, except for troubleshooting purposes.

**Comment by rijkvanzanten** [comment]
You could set `LOG_LEVEL=warn` to only show warnings and more severe log items 🤔

**Comment by duel-adn** [comment]
Yes, but in such a case I'm going to miss other messages that can be useful

**Reply by rijkvanzanten** [reply]
Just realized a better solution:

You can actually use `LOGGER_HTTP_*` to configure the pino-http logger, which in turn means you can use `LOGGER_HTTP_USE_LEVEL=trace` to use the `trace` log level for the requests themselves 🙂

**Comment by duel-adn** [comment]
Thanks for the hint.
Just in case, the following is the configuration I used in the .env file:

`LOGGER_HTTP_AUTO_LOGGING__IGNORE_PATHS=["/server/ping"]`
